Bottom: 74a8d034bc75944533d759e99d4e586b8f9ab994
Top:    f85472204a6a85e02077990998033f19acba93fd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 22:54:53 +0100

Refresh of rename-hidecrosshair-and-resto

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 06226d2..9834577 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -755,26 +755,6 @@ notify_crosshair_change (bool changes_complete)
     }
 }
 
-/* ---------------------------------------------------------------------------
- * notify the GUI that data relating to the mark is being changed.
- *
- * The argument passed is false to notify "changes are about to happen",
- * and true to notify "changes have finished".
- *
- * Each call with a 'false' parameter must be matched with a following one
- * with a 'true' parameter. Unmatched 'true' calls are currently not permitted,
- * but might be allowed in the future.
- *
- * GUIs should not complain if they receive extra calls with 'true' as parameter.
- * They should initiate a redraw of the mark - which may (if necessary) mean
- * repainting the whole screen if the GUI hasn't tracked the mark's location.
- */
-void
-notify_mark_change (bool changes_complete)
-{
-  /* For now, just piggy back on the crosshair changes redrawing the mark */
-  notify_crosshair_change (changes_complete);
-}
 
 /* ---------------------------------------------------------------------------
  * switches crosshair on
@@ -826,7 +806,6 @@ HideCrosshair (void)
     }
 
   notify_crosshair_change (false);
-  notify_mark_change (false);
 }
 
 void
@@ -841,7 +820,6 @@ RestoreCrosshair (void)
     }
 
   notify_crosshair_change (true);
-  notify_mark_change (true);
 }
 
 /* ---------------------------------------------------------------------------
